apps/storybook/docs/d_planning/3_2024-van-v1-naar-v2-beschikbaar.stories.mdx

```diff
@@ -45,6 +45,29 @@ beschikbaar is (zal zijn), je kan doorklikken op de naam om naar de documentatie
 
 ## Geïmpacteerde Web Componenten
 
+### Body
+
+Wordt niet meer voorzien in v2.
+
+<table  style={{width: 100 + '%'}}>
+    <tr>
+        <th>Naam</th>
+        <th>Toestand</th>
+        <th>v1 - Artifact</th>
+        <th>v1 - Gebruik</th>
+        <th>v2 - Artifact</th>
+        <th>v2 - Gebruik</th>
+    </tr>
+    <tr>
+        <td>[Body [legacy]](/docs/elements-body--documentatie)</td>
+        <td>deprecated</td>
+        <td>elements</td>
+        <td><code>{`<body is="vl-body">`}</code></td>
+        <td>-</td>
+        <td>-</td>
+    </tr>
+</table>
+
```

Comment:
conform de form ook de Body [next] tabel voorzien waarin staat dat je de native 'body' hoort te gebruiken; zodat duidelijk is wat dan wel de bedoeling is